data-view.css

```diff
@@ -1,48 +1,57 @@
 /* styles.css */
 body {
```

Comment:
use Boxing Sizing for better Responsive experince

Reply:
fix